modules/indexer/issues/indexer.go

```diff
@@ -102,6 +104,13 @@ var (
 func InitIssueIndexer(syncReindex bool) {
 	waitChannel := make(chan time.Duration)
 
+	if setting.Indexer.IssueType != "bleve" {
+		// When bleve isn't used, shutdown any open goroutines they have.
+		bleve.Config.Shutdown()
```

Comment:
we did not do that atm - is bleve already initialized at this point ?!?

Reply:
Bleve is initialized by the init functions, so it is initialized at this point.

Reply:
See the ref in OP comment. That's the init function.